appveyor.yml

```diff
@@ -125,6 +125,9 @@ install:
   - copy C:\Python27\python.exe C:\Python27\python2.7.exe
   - set PATH=C:\Python27;%PATH%
 
+  # Run unit tests for bootstrap.py
+  - python src\bootstrap\bootstrap_test.py
```

Comment:
@aidanhs I just thought it was a good place because the 2 preconditions that I need are already meet at this stage:

* [x] The repository is already cloned
* [x] Python is already configured

But I'm open to suggestions, please let me know where do you think is the right place to move this section.